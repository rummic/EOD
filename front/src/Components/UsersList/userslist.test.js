import React from "react";
import { create } from "react-test-renderer";
import UserList from "./userslist";
import { shallow } from 'enzyme';

describe('userslist', () => {
    test('Rendering userslist component', () => {
        const wrapper = shallow(<UserList />);
        expect(wrapper).toMatchSnapshot();
    });
});  


/*
    describe("userslist component", () => {
    it("shows a list of userslist", async () => {
    const component = create(<userslist />);
    const instance = component.getInstance();
    await instance.fetch("https://localhost:44388/api/Users");
    console.log(instance.state);
    });
*/
/*
  fetch("https://localhost:44388/api/Users", {
    method: "GET",
    headers: {
      "Content-Type": "application/json",
      Authorization: `bearer ${token}`
    }
  }).then(response =>
    response.json().then(responseJSON => {
      console.log(responseJSON);
      this.setState({
        users: responseJSON.value || []
      });
    })
  );
*/

