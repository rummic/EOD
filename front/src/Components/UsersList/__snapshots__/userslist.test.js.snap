// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`userslist Rendering userslist component 1`] = `ShallowWrapper {}`;
